src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,45 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int length = 10;
+    private K[] keys;
+    private V[] values;
+    private int size;
+
+    public StorageImpl() {
+        keys = (K[]) new Object[length];
+        values = (V[]) new Object[length];
+        size = 0;
+    }
```

Comment:
```suggestion
    public StorageImpl() {
        keys = (K[]) new Object[length];
        values = (V[]) new Object[length];
    }
```

переменная size инициализируется нулем по умолчанию

Reply:
согласна с комментария. просто не поняла что делать. теперь исправила,